package.json

```diff
@@ -9,7 +9,7 @@
   },
   "main": "dist/npm/index.js",
   "module": "dist/es/index.js",
-  "browser": "dist/streaming-client.browser.js",
+  "web": "dist/streaming-client.browser.js",
```

Comment:
`"web"` does not have any significance for build tools. I renamed it to this just so you can quickly see what the file name and purpose of the file is.